source/JsMaterialX/JsMaterialXCore/JsUtil.cpp

```diff
@@ -1,53 +1,38 @@
-#include "../helpers.h"
-#include <MaterialXCore/Node.h>
-#include <MaterialXCore/Util.h>
+#include "../vectorHelper.h"
 
-#include <MaterialXFormat/File.h>
-#include <iostream>
-#include <vector>
-#include <iterator>
-#include <map>
+#include <MaterialXCore/Util.h>
 
-#include <emscripten.h>
 #include <emscripten/bind.h>
 
 namespace ems = emscripten;
 namespace mx = MaterialX;
 
 extern "C"
 {
-    EMSCRIPTEN_BINDINGS(my_module)
+    EMSCRIPTEN_BINDINGS(util)
     {
+        ems::constant("EMPTY_STRING", mx::EMPTY_STRING);
+
         ems::function("getVersionString", &mx::getVersionString);
-        ems::function("createValidName", &mx::createValidName); // arg0 === {std::string}, arg1 === {unicode representing character}
-        ems::function("isValidName", &mx::isValidName);
-        ems::function("incrementName", &mx::incrementName);
 
-        // The following function throw: Cannot call {function name} due to unbound types: XXXXX
         ems::function("getVersionIntegers", ems::optional_override([]() {
-                     std::tuple<int, int, int> version = mx::getVersionIntegers();
-                     return arrayToVec((int *)&version, 3);
+                     auto version = mx::getVersionIntegers();
```

Comment:
I would still prefer to make the `std::tuple<int, int, int>` explicit, since the pointer arithmetic below depends on the number of elements. With `auto`, it's not obvious how many elements need to be dealt with, and more importantly, it won't fail at compilation time if the number of elements (i.e. return type of the function) ever changes. 
This goes for all `getVersionIntegers` and related functions where we apply this wrapping.